README.md

```diff
@@ -27,8 +27,8 @@ make -j4
 
 ## Build Linux version in docker
 ```
-docker build -t updater2 .
-docker run -v `pwd`/build-docker:/build-docker -u `id -u $USER` updater2
+docker build -t unvupd-linux .
```

Comment:
name is kinda meh. UnvanquishedLauncher or something is better. unvlauncher if we want to keep it shorter?

Reply:
All right, I put unvlauncher.